config/arrays/wiki/defaults.yml

```diff
@@ -2,12 +2,12 @@ Compare: "True"
 Player: Event Player
 Primary Fire: Button(Primary Fire)
 Vector: Vector(0, 0, 0)
-Number: 1
+Number: 0
+Number (1): 1
```

Comment:
Kind of ugly, but it works.

Reply:
yeah, some values really should default to 0, like the index for loops. others really shouldnt, like the incremented number or chase rate. not sure what else would be a better solution